### DISCUSSION TITLE
System.ArgumentException: The item HUD already exists in this Dropdown. when opening skin editor while spectating multiplayer

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#23869] [Q&A] [answered]
![asdfasd](https://github.com/ppy/osu/assets/101510334/efd80201-7bbf-4ea5-8365-0e39c2b6fc4d)

# Version
 https://github.com/ppy/osu/releases/tag/2023.610.0

# Logs

[runtime.log](https://github.com/ppy/osu/files/11713191/runtime.log)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/23886
